sky/tools/create_macos_framework.py

```diff
@@ -71,29 +72,6 @@ def main():
   return 0
 
 
-def process_framework(dst, args, framework_path):
-  framework_binary = sky_utils.get_mac_framework_dylib_path(framework_path)
-
-  if args.dsym:
```

Comment:
Did the `args.dsym` check move somewhere, to avoid the debug and profile versions from generating it?

Reply:
Yep -- it took a slightly different form; added a comment where.